shared/common-adapters/switch.tsx

```diff
@@ -41,9 +42,11 @@ const LabelContainer = props =>
       {props.children}
     </Kb.WithTooltip>
   ) : (
-    <Kb.Box2 direction="vertical" style={styles.labelContainer}>
-      {props.children}
-    </Kb.Box2>
+    <Kb.ClickableBox onClick={props.allowLabelClick ? props.onClick : undefined}>
```

Comment:
Need an `alignSelf: 'center'` on this, `Box2` adds it internally

Reply:
fixed by swapping order of boxes